src/net/sourceforge/kolmafia/session/BadMoonManager.java

```diff
@@ -7,41 +7,41 @@
 import net.sourceforge.kolmafia.preferences.Preferences;
 
 public abstract class BadMoonManager {
-  public static final String STAT1 = "+20 to one stat, -5 to others";
-  public static final String STAT2 = "+40 to one stat, -50% Familiar Weight";
-  public static final String STAT3 = "+50% to one stat, -50% to another";
-  public static final String DAMAGE1 = "+10 damage, Damage Reduction -2";
-  public static final String DAMAGE2 = "+20 damage, 1-3 damage/round to self";
-  public static final String RESIST1 = "So-So resistance to one, Vulnerability to opposites";
-  public static final String RESIST2 = "Resistance to all, -attributes";
-  public static final String ITEM_DROP = "Item Drop";
-  public static final String MEAT_DROP = "Meat Drop";
-  public static final String DAMAGE_REDUCTION = "+ Damage Reduction, - Weapon Damage";
-  public static final String MEAT = "Meat";
-  public static final String ITEMS = "Items";
-
-  public static final String[] TYPES = {
-    STAT1,
-    STAT2,
-    STAT3,
-    DAMAGE1,
-    DAMAGE2,
-    RESIST1,
-    RESIST2,
-    ITEM_DROP,
-    MEAT_DROP,
-    DAMAGE_REDUCTION,
-    MEAT,
-    ITEMS,
-  };
+  public enum Type {
```

Comment:
Whenever (somebody) converted the class(?) or Object array(?) into "record Encounter", they declared a field named "type" for the "type of the encounter".

I sort of wish they'd named it "encounterType" - in which case this enum could have been "EncounterType".

Which is a lot more informative than a top-level enum named "Type".

This is named consistently with the existing code, so I will accept it - but I will still grumble about the existing code. :)

Reply:
it could well be `EncounterType`, but in this case as it's an inner enum, I see it as being called `BadMoonManager.Type` -- perhaps not entirely descriptive, but you can infer what it means.